CMakeLists.txt

```diff
@@ -94,7 +94,7 @@ if(ITKPythonPackage_SUPERBUILD)
   set(ITK_REPOSITORY "https://github.com/InsightSoftwareConsortium/ITK.git")
 
   # ITK release 2021-10-27
-  set(ITK_GIT_TAG "30cd5fa095a6b2e52fac052e4a84bb3740fcc2b1")
+  set(ITK_GIT_TAG "4c25e667a5")
```

Comment:
It is advisable to put the full sha1 here.

Reply:
That's fine, was copying from `master`. Will fix.